tests/nodeos_snapshot_diff_test.py

```diff
@@ -213,7 +213,8 @@ def waitForBlock(node, blockNum, blockType=BlockType.head, timeout=None, reportI
     Utils.processLeapUtilCmd("snapshot to-json --input-file {}".format(progSnapshotFile), "snapshot to-json", silentErrors=False)
     progSnapshotFile = progSnapshotFile + ".json"
 
-    assert Utils.compareFiles(progSnapshotFile, irrSnapshotFile), f"Snapshot files differ {progSnapshotFile} != {irrSnapshotFile}"
+    # you cant compare prog to irr as it can be off for 1 block since it was scheduled to be executed "asap"
```

Comment:
Instead can you use the scheduling to schedule the one  you want so you can compare.